### ISSUE TITLE
Text reverts to default height when using parametric text

### ISSUE THREAD

**Issue by npolanosky** [thomasa88/ParametricText#21]

**Comment by thomasa88** [comment]
Hi, I cannot reproduce this issue.

I changed text size to 30 mm (from 12) and set bold and italic. All properties remained after applying parametric text.

Fusion version: 2.0.10148

Video:
https://user-images.githubusercontent.com/89997/120063334-ebc9cc00-c066-11eb-8fb5-0e3dff43bd95.mp4

**Comment by thomasa88** [comment]
I can reproduce this issue in Fusion 360 v. 2.0.10244.

It applies to text that is not on a path as well.

**Comment by thomasa88** [comment]
Likely related to this new feature..
"Improved! Text Height is now an editable parameter", https://www.autodesk.com/products/fusion-360/blog/may-2021-product-update-whats-new/ .

**Comment by thomasa88** [comment]
Autodesk has broken the SketchText API once again.

Reported upstream: https://forums.autodesk.com/t5/fusion-360-api-and-scripts/bug-setting-sketchtext-properties-resets-text-height-since-v-2-0/td-p/10356396

I tried a workaround of storing the text height, changing the text and then writing the height back, but any change to any property of the text leads to the height being reset to 10 mm (1.0 cm).

Only workaround is probably to block Fusion 360 from updating or storing away a good version of Fusion 360 before it updates.. But using an old version might mean it stops working with the cloud.

```
app = adsk.core.Application.get() 
t = app.activeProduct.rootComponent.sketches[0].sketchTexts[0]  

t.height 
3.0

t.text="new" 

t.height 
1.0

t.height=3.0

t.height 
1.0
```

**Comment by thomasa88** [comment]
A workaround is now available in 2.2.0-beta.1.

Please try it out and report if it works for you.

Make sure to remove any version installed from the app store before installing a version from github.
Direct download: https://github.com/thomasa88/ParametricText/releases/download/v2.2.0-beta.1/ParametricText-v2.2.0-beta.1.zip

Release page: https://github.com/thomasa88/ParametricText/releases/tag/v2.2.0-beta.1

Installation instructions: https://github.com/thomasa88/ParametricText/blob/v2.2.0-beta.1/README.md#installation

**Comment by thomasa88** [comment]
@Casella75  @npolanosky 

I did some tweaks to beta.1 and have now uploaded beta.2. Please see the links in the comment above.
